test/web-platform-tests/to-upstream/html/semantics/embedded-content/the-img-element/remove-attribute-src.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Disabled checkbox event handling</title>
+<link rel="author" title="atsikov" href="mailto:alexey.tsikov@gmail.com">
+<link rel=help href="https://html.spec.whatwg.org/#the-img-element">
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<body>
+<script>
+"use strict";
+
+function testRemoveSrc(image, t) {
+  image.removeAttribute("src");
+  assert_equals(image.width, 0);
+  assert_equals(image.height, 0);
+  assert_equals(image.complete, false);
+  assert_equals(image.src, "");
+  assert_equals(image.currentSrc, "");
+  t.done();
+}
+
+async_test(t => {
+  const image = new window.Image();
+  image.onload = () => {
+    image.onload = () => {
```

Comment:
You can make this a bit shorter: `image.onload = t.unreached_func("onload should not be called...");`.